src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/health/health.component.html

```diff
@@ -1,164 +1,240 @@
-                             (prepareFn)="preparePoolUsage($event[0], $event[1])"></cd-health-pie>
-            </div>
-          </td>
-        </tr>
-        <tr>
-          <td i18n>Objects</td>
-          <td>
-            <ng-container i18n>Raw capacity</ng-container>
-            <br>
-            <ng-container i18n="disk used">({{ contentData.df.stats.total_used_bytes | dimlessBinary }} used)</ng-container>
-          </td>
-          <td i18n>Usage by pool</td>
-        </tr>
-      </table>
+    <cd-info-card cardTitle="Monitors"
+                  link="/monitor"
+                  class="col-sm-6 col-md-4 col-lg-3"
+                  contentClass="content-highlight"
+                  *ngIf="contentData.mon_status">
+      {{ contentData.mon_status | monSummary }}
+    </cd-info-card>
+
+    <cd-info-card cardTitle="OSDs"
+                  link="/osd"
+                  class="col-sm-6 col-md-4 col-lg-3"
+                  contentClass="content-highlight"
+                  *ngIf="contentData.osd_map">
+      <span *ngFor="let result of (contentData.osd_map | osdSummary)" [ngStyle]="result.style">
+        {{ result.content }}
+      </span>
+    </cd-info-card>
+
+    <cd-info-card cardTitle="Manager Daemons"
+                  class="col-sm-6 col-md-4 col-lg-3"
+                  contentClass="content-highlight"
+                  *ngIf="contentData.mgr_map">
+      {{ contentData.mgr_map | mgrSummary }}
+    </cd-info-card>
+
+    <cd-info-card cardTitle="Hosts"
+                  link="/hosts"
+                  class="col-sm-6 col-md-4 col-lg-3"
+                  contentClass="content-medium content-highlight">
+      {{ contentData.hosts }}
+    </cd-info-card>
+
+    <cd-info-card cardTitle="Object Gateways"
+                  link="/rgw/daemon"
+                  class="col-sm-6 col-md-4 col-lg-3"
+                  contentClass="content-medium content-highlight">
+      {{ contentData.rgw }}
+    </cd-info-card>
+    <cd-info-card cardTitle="Metadata Servers"
+                  class="col-sm-6 col-md-4 col-lg-3"
+                  contentClass="content-highlight"
+                  *ngIf="contentData.fs_map">
+      {{ contentData.fs_map | mdsSummary }}
+    </cd-info-card>
+
+    <cd-info-card cardTitle="iSCSI Gateways"
+                  link="/block/iscsi"
+                  class="col-sm-6 col-md-4 col-lg-3"
+                  contentClass="content-medium content-highlight">
+      {{ contentData.iscsi_daemons }}
+    </cd-info-card>
+  </cd-info-group>
+
+  <cd-info-group groupTitle="Performance"
+                 class="row info-group">
+
+    <cd-info-card cardTitle="Client IOPS"
+                  class="cd-col-5"
+                  cardClass="card-medium"
+                  contentClass="content-medium content-highlight"
+                  *ngIf="contentData.client_perf">
+      {{ (contentData.client_perf.read_op_per_sec + contentData.client_perf.write_op_per_sec) | round:1 }}
```

Comment:
As explained in another comment, I created a pipe because of reusability.

Reply:
But it is only used once in the dashboard, I'm not sure it will be used anywhere else as the dimless pipe brings a better usage for the most cases. 

I know you did it because of reusability, but it's only a function call with no additional changes. I'm not in favor maintaining a pipe that just calls a library function.

IMHO it's a really complicated way of just calling `Math.round`. You could just instead call `Math.round(<number>)` inside the template without any changes inside the component.